### PR TITLE
optv - 0.2.4

### PR DESCRIPTION
**Pull request by alexlib**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by alexlib** [comment]
@conda-forge-admin, please ping team

**Comment by chrisburr** [comment]
@conda-forge-admin, please restart ci

**Comment by beckermr** [comment]
This may need a rebase.

**Comment by beckermr** [comment]
Looks like the build tests are failing @alexlib. I cannot figure out why exactly. They appear to be missing files and there also appear to be missing attributes on some of the objects. It is probably best for you to test the build locally on your machine and diagnose the problem there.

File: recipes/liboptv/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  name: optv
+  version: {{ version }} 
+
+source:
+  url: https://github.com/OpenPTV/openptv/archive/{{ version }}.tar.gz
+  sha256: 4cafc70d966f2694cc936973eb2581d35bdffd6ec4b30f9ff99a1515c07f4042
+
+build:
+  number: 0
+  skip: True  # [not py3k]
```
Comment (chrisburr) [inline recipes/liboptv/meta.yaml]: Looking at the failures I think this was intended to be?
```suggestion
  skip: True  # [py3k]
```

File: recipes/liboptv/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  - {{ PYTHON }} setup.py install
+  
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - pip
+    - numpy 1.16.1
```
Comment (chrisburr) [inline recipes/liboptv/meta.yaml]: It's probably safer not to specify a minimum patch version
```suggestion
    - numpy 1.16
```

**Pull request closed without merging**